app/routes/joblistings/components/JoblistingEditor.tsx

```diff
@@ -223,16 +224,7 @@ const JoblistingEditor = () => {
               component={SelectInput.AutocompleteField}
               filter={['companies.company']}
               onChange={(event) => {
-                fetchContacts(event).then(() => {
-                  form.change('joblistingEditor', {
-                    label: 'Ingen',
-                    value: null,
-                  });
-                  form.change('responsible', {
-                    label: 'Ingen',
-                    value: null,
-                  });
```

Comment:
No idea why this was here

Reply:
I think we need the 'responsible' one. If you change the company, there will be a different set of responsible persons (Kontaktpersoner) to choose from. So if we have selected a responsible person from company A and then change to company B we need to clear the clear the field to not end up with an invalid responsible person.

Reply:
ah yes that actually makes sense haha